anta/tests/connectivity.py

```diff
@@ -107,22 +107,38 @@ class Neighbor(BaseModel):
     @AntaTest.anta_test
     def test(self) -> None:
         """Main test function for VerifyLLDPNeighbors."""
-        command_output = self.instance_commands[0].json_output
-
+        failure_type_map = {
+            "port_not_configured": "Port(s) not configured",
+            "no_lldp_neighbor": "No LLDP neighbor(s)",
+            "wrong_lldp_neighbor": "Wrong LLDP neighbor(s)",
+        }
```

Comment:
maybe we could move this to a test function called "generate_failure_message" taking the requires inputs